instrumentation/aws-sdk/aws-sdk-2.2/javaagent/aws-sdk-2.2-javaagent.gradle

```
@@ -5,6 +5,9 @@ muzzle {
     group = "software.amazon.awssdk"
     module = "aws-core"
     versions = "[2.2.0,)"
+    // Used by all SDK services, the only case it isn't is an SDK extension such as a custom HTTP
+    // client, which is not target of instrumentation anyways.
+    additionalDependencies = ["software.amazon.awssdk:protocol-core"]
```

I think that we should use the `extraDependency` syntax that's being used everywhere else

```suggestion
        extraDependency "software.amazon.awssdk:protocol-core"
```

---

Oops